vtr_flow/tasks/regression_tests/vtr_reg_basic/hdl_include_odin/config/config.txt

```diff
@@ -0,0 +1,41 @@
+#################################################################
+# Configuration file for running experiments                    #
+#                                                               #
+# This config file is testing the ability to specify include    #
+# files that should pass to the VTR frontend with the top       #
+# module of the benchmark (ch_intrinsic_top.v). This is done    #
+# by specifying two Verilog header files that provide essential # 
+# definitions, and memory_controller design that provides the   #
+# design of an internal component for ch_intrinsic_top. If the  #
```

Comment:
Is ch_intrinsic_top or ch_intrinsics_modified the top level module?  (Not sure: update comment if the latter)